src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -1611,33 +1611,15 @@ class RideConstructionWindow final : public Window
         widgets[WIDX_RIGHT_CURVE_SMALL].type = WindowWidgetType::Empty;
         widgets[WIDX_LEFT_CURVE_VERY_SMALL].type = WindowWidgetType::Empty;
         widgets[WIDX_RIGHT_CURVE_VERY_SMALL].type = WindowWidgetType::Empty;
-        widgets[WIDX_LEFT_CURVE_SMALL].left = 28;
-        widgets[WIDX_LEFT_CURVE_SMALL].right = 49;
-        widgets[WIDX_RIGHT_CURVE_SMALL].left = 116;
-        widgets[WIDX_RIGHT_CURVE_SMALL].right = 137;
-        widgets[WIDX_LEFT_CURVE_SMALL].image = ImageId(SPR_RIDE_CONSTRUCTION_LEFT_CURVE);
-        widgets[WIDX_RIGHT_CURVE_SMALL].image = ImageId(SPR_RIDE_CONSTRUCTION_RIGHT_CURVE);
         if (IsTrackEnabled(TRACK_CURVE_VERTICAL))
         {
             widgets[WIDX_LEFT_CURVE_SMALL].type = WindowWidgetType::FlatBtn;
-            widgets[WIDX_LEFT_CURVE_SMALL].left = 6;
-            widgets[WIDX_LEFT_CURVE_SMALL].right = 27;
-            widgets[WIDX_LEFT_CURVE_SMALL].image = ImageId(SPR_RIDE_CONSTRUCTION_LEFT_CURVE_SMALL);
             widgets[WIDX_RIGHT_CURVE_SMALL].type = WindowWidgetType::FlatBtn;
-            widgets[WIDX_RIGHT_CURVE_SMALL].left = 138;
-            widgets[WIDX_RIGHT_CURVE_SMALL].right = 159;
-            widgets[WIDX_RIGHT_CURVE_SMALL].image = ImageId(SPR_RIDE_CONSTRUCTION_RIGHT_CURVE_SMALL);
         }
         if (IsTrackEnabled(TRACK_CURVE))
         {
             widgets[WIDX_LEFT_CURVE].type = WindowWidgetType::FlatBtn;
             widgets[WIDX_RIGHT_CURVE].type = WindowWidgetType::FlatBtn;
-            widgets[WIDX_LEFT_CURVE_SMALL].left = 6;
-            widgets[WIDX_LEFT_CURVE_SMALL].right = 27;
-            widgets[WIDX_LEFT_CURVE_SMALL].image = ImageId(SPR_RIDE_CONSTRUCTION_LEFT_CURVE_SMALL);
-            widgets[WIDX_RIGHT_CURVE_SMALL].left = 138;
-            widgets[WIDX_RIGHT_CURVE_SMALL].right = 159;
-            widgets[WIDX_RIGHT_CURVE_SMALL].image = ImageId(SPR_RIDE_CONSTRUCTION_RIGHT_CURVE_SMALL);
```

Comment:
I'm not sure I follow this part, what it was doing was to replace the small curve icons by maybe not a small curve when the ride didn't had one, so as to not increase the width of the coaster?

Reply:
It previously only had space for 7 buttons side by side (3 types of curve, both left and right, plus straight on), so it had to rearrange the placement. Now that the UI is wide enough to display all 9 buttons necessary, it doesn’t need to change positions any more.